### PR TITLE
Add .editorconfig file

### PR DESCRIPTION
**Pull request by majecty**
I copied the file from CodeChain repository

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+indent_style=space
+indent_size=4
+tab_width=8
+end_of_line=lf
+charset=utf-8
+trim_trailing_whitespace=true
+max_line_length=120
+insert_final_newline=true
+
+[*.yml]
```
Comment (sgkim126) [inline .editorconfig]: I think this part is not necessary for this repository.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+[.github/**/*.yml]
+indent_style=space
+indent_size=2
+tab_width=8
+end_of_line=lf
+charset=utf-8
+trim_trailing_whitespace=true
+insert_final_newline=true
+
+[.travis.yml]
```
Comment (sgkim126) [inline .editorconfig]: ditto

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+indent_style=space
+indent_size=2
+tab_width=8
+end_of_line=lf
+charset=utf-8
+
+[*.json]
```
Comment (sgkim126) [inline .editorconfig]: ditto

**Review by sgkim126 — CHANGES_REQUESTED** [review]

**Comment by majecty** [comment]
@sgkim126 I updated!

**Pull request merged**